Base/Logic/vtkSystemInformation.cxx

```diff
@@ -238,7 +238,7 @@ void vtkSystemInformation::RunRenderingCheck()
   this->RenderingCapabilities |= ONSCREEN_RENDERING;
 #elif defined(VTK_USE_X)
   // if using X, need to check if display is accessible.
-  Display* dId = XOpenDisplay((char*)NULL);
+  Display* dId = XOpenDisplay((char*)nullptr);
```

Comment:
Is the `(char*)` needed ?

Reply:
It is casting it to a pointer so I don't think is needed.

Reply:
Since we will use commit like this one as example for updating a lot more code (e.g extension), could this be clarified ?

Reply:
Replaced to just nullptr. Updated commit message.

Reply:
Reference:
- https://stackoverflow.com/questions/45002305/whats-the-difference-between-xopendisplay0-and-xopendisplaynull
- https://stackoverflow.com/questions/4705968/what-is-value-of-eof-and-0-in-c/4706217#4706217
- https://stackoverflow.com/questions/15954506/how-can-a-char-variable-accept-pointernull-as-its-value